src/components/config/i18next.js

```diff
@@ -0,0 +1,29 @@
+import i18n from "i18next";
+import { initReactI18next } from "react-i18next";
+import translationAR from "../locales/ar/translation.json";
+import translationTR from "../locales/tr/translation.json";
```

Comment:
hmm why are these files here too in the searchbox one? maybe you need to pull?

Reply:
I did pull before pushing this, that's why it came here too I believe. I did pull and solve conflicts so that you can directly merge to `dev` if everything is okay.